packages/plugins/typescript/react-apollo/src/visitor.ts

```diff
@@ -174,7 +174,7 @@ export class ReactApolloVisitor extends ClientSideBaseVisitor<ReactApolloRawPlug
 
     if (operationType === 'Query') {
       const lazyOperationName: string = this.convertName(node.name.value, {
-        suffix: titleCase('LazyQuery'),
+        suffix: pascalCase('LazyQuery'),
```

Comment:
I've kept this call here even though it's an expensive no-op to aid future copy-pasting. We could simplify this to:

```suggestion
        suffix: 'LazyQuery',
```